src/Game/Game.cpp

```diff
@@ -448,5 +446,19 @@ Audio::Mixer* Game::mixer()
     return _mixer.get();
 }
 
+Event::Dispatcher& Game::eventDispatcher()
+{
+    return _eventDispatcher;
+}
+
+}
+
+namespace Event
+{
+// static
+Dispatcher* Dispatcher::getInstance()
+{
+    return &Game::getInstance()->eventDispatcher();
+}
```

Comment:
Why can't you make Dispatcher a singleton of it's own? Otherwise you can just add it as Game property. Singletons are hideous creatures when misused.